src/bootstrap/doc.rs

```diff
@@ -597,6 +597,9 @@ fn doc_std(
             .arg("--resource-suffix")
             .arg(&builder.version)
             .args(extra_args);
+        if builder.config.library_docs_private_items {
+            cargo.arg("--document-private-items");
```

Comment:
Might want to add `--document-hidden-items` too here since some items are unstable + hidden instead of private due to the core/std split.

Reply:
Good point, done!